cristin-commons/src/main/java/no/unit/nva/cristin/model/SearchResponse.java

```diff
@@ -226,10 +226,9 @@ public boolean equals(Object o) {
         if (this == o) {
             return true;
         }
-        if (!(o instanceof SearchResponse)) {
+        if (!(o instanceof SearchResponse<?> that)) {
```

Comment:
E or ? here?

Reply:
? means any class, not known.
E means a spesific Class not known here.